ui/app/templates/workspace/projects/index.hbs

```diff
@@ -18,6 +18,10 @@
       {{/if}}
     </small>
   </div>
+  <LinkTo @route="workspace.projects.new" class="pds-button pds--primary pds--iconStart">
+    <Pds::Icon @type="plus-plain" class="pds-button__iconStart"/>
+    &nbsp;New project
```

Comment:
Heh, the `&nbsp` here makes me think we need to handle more cases in strucutre, I dont' think this is the sort of thing you should have to do.

Also if you wanted, there's `Pds::CtaLink` that wraps `LinkTo` but is styled more like a button. Not sure if it will be a drop in here, but may be worth checking out.

Reply:
Just used the `Pds::CtaLink`, worked perfectly!